apps/src/p5lab/spritelab/defaultSprites.json

```diff
@@ -703,6 +712,195 @@
       "looping": true,
       "frameDelay": 2,
       "version": "RV2Re1P6P8WVQ_g_HytRqSBokiQG1uUZ"
+    },
+    "category_backgrounds/background_cave": {
```

Comment:
we should separate the backgrounds into their own similarly-formatted `defaultBackgrounds.json` file. we'll have some repetitive code for pre-loading and storing animations in redux, but i think it will be clearer that they server different purposes

Reply:
talked about this in our 1:1 but we decided not to do this